.github/ISSUE_TEMPLATE/new-feature.md

```diff
@@ -0,0 +1,37 @@
+---
+name: New Feature
+about: Issues for requesting new features on the bot.
+title: ''
+labels: 'status: planning, type: feature'
+assignees: ''
+
+---
+
+## Description
+<!-- Describe the feature you'd like to see added -->
+
+
+## Reasoning
+<!-- Why would the feature be a useful addition to the community -->
+
+
+## Proposed Implementation
+<!-- If you had a way to implement the feature in mind, describe it here, including any command signatures and concept screenshots if possible -->
+
+
+## Possible Alternatives
```

Comment:
I know this is optional but it seems like it's getting a bit overwhelming with this section. Would it be better to just let this be part of "Additional Details"?

Reply:
It could be removed, sure. I added it mostly because I've noticed newer members have a tendency to not bother considering alternate implementations, or stating they've discussed alternatives that were turned down for whatever reason. Without a specific section, those specific people won't bother to even think about it.

Reply:
They could be mentioned in the comments of the "Additional Details".

Reply:
That's not a bad idea actually.